internal/kafka/test/integration/ocm_client_test.go

```diff
@@ -0,0 +1,79 @@
+package integration
+
+import (
+	"testing"
+
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/kafka/test"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/client/ocm"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/test/mocks"
+	"github.com/onsi/gomega"
+	amsv1 "github.com/openshift-online/ocm-sdk-go/accountsmgmt/v1"
+)
+
+func TestOCMClientGetQuotaCosts(t *testing.T) {
```

Comment:
@miguelsorianod had to do an integration test to test the get quota cost logic because I couldn't mock the ocm client itself.